Makefile

```diff
@@ -2,7 +2,8 @@
 
 .PHONY: prepare build clean docker run
 
-GO=CGO_ENABLED=0 GOOS=linux go
+GO=CGO_ENABLED=0 GO111MODULE=on go
+GOCGO=CGO_ENABLED=1 GO111MODULE=on go
```

Comment:
Looks like this was copied from edgex-go. I don't think you need the second line since I don't believe anything in this service requires CGO compilation. Also on the first line I notice the `GOOS=linux go` has been removed. Was that on purpose?

Reply:
@tsconn23 ok, yes, I copied those two lines from edgex-go. We can keep GOOS=linux flag- it is just that originally I thought it does not have to be limited to linux.

Reply:
@tsconn23 Why do we have GOOS=linux for this one?

Reply:
Same question originally here: My vote is to remove that limitation on GOOS=linux since we should also be able to compile the golnag code on Windows as well.  Also, as I know, Vault can run on Windows, too.

Reply:
@tingyuz needs to answer this. I have no idea.

Reply:
The purpose is to build the binary for Linux platform only as the result of the step ( build/docker/run) in the make file are targeting Linux only.

Reply:
ok, it looks like the original intention of this service is for Linux platform only, then I'll keep that linux GOOS flag. - Jim Wang